modules/cli/src/main/scala/scala/cli/commands/publish/PublishConnectionOptions.scala

```diff
@@ -0,0 +1,45 @@
+package scala.cli.commands.publish
+
+import caseapp.*
+
+import scala.cli.commands.shared.{HelpGroup, SharedVersionOptions}
+import scala.cli.commands.tags
+
+// format: off
+final case class PublishConnectionOptions(
+  @Group(HelpGroup.Publishing.toString)
+  @HelpMessage("Connection timeout, in seconds.")
+  @Tag(tags.restricted)
```

Comment:
Shouldn't all publish-related options be experimental?
```suggestion
  @Tag(tags.experimental)
```

Reply:
None of them are and I gues it's to not dispaly more `experimental` warnings than necessary.
Only warnings about using the command and all publish-related using directives are displayed right now.
Should we change it?

Reply:
tbh it feels like we should, if only to be consistent...
doesn't make sense to have `restricted` options for an `experimental` command

Reply:
Let's get into it in another PR